services/frontend-service/pkg/handler/handle_test.go

```diff
@@ -2109,3 +2109,102 @@ func TestServer_HandleAAEnvironments(t *testing.T) {
 		})
 	}
 }
+
+func TestServer_HandleDeleteAAEnvConfig(t *testing.T) {
+	tests := []struct {
+		name                 string
+		req                  *http.Request
+		batchResponse        *api.BatchResponse
+		expectedResp         *http.Response
+		expectedBody         string
```

Comment:
Why is `expectedBody` not part of `expectedResp`?

Either move body to be a part of the response - or change `expectedResp` to be an int, because the status code is the only thing we check for anyway.